STYLE-GUIDE.md

```diff
@@ -2,22 +2,41 @@
 
 This document acts as a Style Guide for the language and wording used in exercises.
 
-## General principle
+## Application
 
-All content should be written in US English.
+These rules should be followed in all exercises. 
+Existing descriptions and test-cases can be updated to adhere to them without requiring "replacement" test cases.
+
+### Consistency within an exercise.
+
+There are some terms that have multiple valid spellings (e.g. "lower case" vs "lowercase"). 
+Where a consistent style has not been agreed within this document, these must be consistent within an exercise.
+
+### Exceptions
+
+Exercises may vary from these rules if there is general consensus across maintainers that this is reasonable. 
+For example, an exercise about converting between different units might choose not to use SI measurements.
+
+## Language
+
+All content should be written in US English. In the future other translations may occur, but the "official" Exercism language is US English.
+
+### Measurements
 All units of measurement must be [SI](https://en.wikipedia.org/wiki/International_System_of_Units) or [SI-derived](https://en.wikipedia.org/wiki/SI_derived_unit) units.
-In the future other translations may occur, but the "official" Exercism language is US English.
 
-## Explain or substitute mathematical and esoteric terms.
+### Abbreviations, acronyms and initialisms
+> All abbreviations, acronyms and initialisms should be defined. 
+> - If the unabbreviated term is clear, always include the full wording at first use (e.g. "Domain-specific language (DSL)");
+> - If the term needs more explaination to be clear to someone unfamiliar, explain it: (e.g. "The four basic types of SQL commands are Create, Read, Update, Delete. The initials of these come together to form the word CRUD.");
+> - If explaining the acryonm adds no tangible value to the student, simply introduce the acryonm as if it were a "proper noun" (e.g. "People's inherit DNA, a set of chemical instructions that influence how their bodies are constructed")
```

Comment:
```suggestion
> - If explaining the acryonm adds no tangible value to the student, simply introduce the acryonm as if it were a "proper noun" (e.g. "People's inherit DNA, a set of chemical instructions that influence how their bodies are constructed")
```

This literally leads to people worrying about that they miss something vital about the exercise and quitting because of it, complaining about it or struggling with it

Reply:
I don't get your comment at all :) You think we should have to write out "deoxyribonucleic acid" rather than just using DNA? I think that just trying to pronounce that in my head would put me off.

Reply:
(see slack for response, then we can come to a clear message and put it here)

Reply:
It should be made clear that _understanding_ what DNA is or what it stands for, isn't important for the exercise or completing it. That should be the rule when something doesn't add tangible value.

It's fine for the story to have "things", but we must make clear that they're not important to understand. Students _will_ spend time looking things up, which is, as you mentioned on slack, a waste.